cluster/nats_rpc_common.go

```diff
@@ -61,11 +62,20 @@ func setupNatsConn(connectString string, appDieChan chan bool, options ...nats.O
 				logger.Log.Errorf(err.Error())
 			}
 		}),
+		nats.ConnectHandler(func(*nats.Conn) {
```

Comment:
Will this `ConnectHandler` only execute the given function if the connection succeeds?

Reply:
Yes